src/fava/util/date.py

```diff
@@ -93,6 +96,31 @@ class Interval(Enum):
     YEAR = "year"
     QUARTER = "quarter"
     MONTH = "month"
+    FORTNIGHT = "fortnight"
+    # Note: While a fortnight is a well-defined time interval, there are no
```

Comment:
Can you move this (maybe in a slightly shortened form) to the help pages this seems relevant for users to understand how this works